stream.go

```diff
@@ -230,10 +230,14 @@ func newClientStream(ctx context.Context, desc *StreamDesc, cc *ClientConn, meth
 	if c.creds != nil {
 		callHdr.Creds = c.creds
 	}
-	var trInfo traceInfo
+	var trInfo *traceInfo
```

Comment:
If I'm not reading this wrong, this variable is only accessed inside the `if` below; then you can remove this line and use `:=` on line 235.

Reply:
No, it's used further down in the call to `newAttemptLocked`.

Reply:
Argh, Chrome ctrl+f fail with dynamically loaded content.